src/hooks/isValidQueryParam.ts

```diff
@@ -0,0 +1,16 @@
+// Use this hook to manipulate incoming or outgoing data.
+// For more information on hooks see: http://docs.feathersjs.com/api/hooks.html
+import { Hook, HookContext } from '@feathersjs/feathers';
+import {paramsSchema} from '../services/users/users.validations';
+import {BadRequest} from '@feathersjs/errors';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export default (options = {}): Hook => {
```

Comment:
Better to create general hook to validate query params and pass schema as options param.

Reply:
Changed to general hook (with parameter -> schema)
ed2fcab9e7527ff7434ea9879bb56714970dc4f7